### PR TITLE
[v6r15] Fixed developer's guide

### PR DESCRIPTION
**Pull request by vipersec**
Added `$DEVROOT` to functions `generateCertificates()` and `generateUserCredentials()` to use the correct directory according to developer's guide.

### CODE REVIEW

**Comment by chaen** [comment]
I understand that you added ```DEVROOT``` for the certificate generation, but why do you want to replace ```SERVERINSTALLDIR``` and the others ?

**Comment by vipersec** [comment]
@chaen I replaced `$SERVERINSTALLDIR` and `$CI_CONFIG` because they are not defined in the guide: http://dirac.readthedocs.io/en/rel-v6r15/DeveloperGuide/DevelopmentEnvironment/DeveloperInstallation/index.html and they were causing problems during the generation of the certificates.

**Comment by atsareg** [comment]
I guess WIP is Work In Progress. Do not forget to change the title when this PR will be ready for merging

**Comment by vipersec** [comment]
I think it's ok now, it will work fine when called from `dirac_ci.sh` and when the `$SERVERINSTALLDIR` is not present (when someone follows the developer's guide).

Feel free to discuss the changes.

File: tests/Jenkins/utilities.sh [hunk 1]
```diff
@@ -2,8 +2,29 @@
...
 ############################################
 
-# Path to ci config files
-CI_CONFIG=$TESTCODE/DIRAC/tests/Jenkins/config/ci
+if [ -z $SERVERINSTALLDIR ]
+then
+	if [ -z $DEVROOT ]
+	then
+		echo 'Environmental variable "DEVROOT" is not set.'
+		read -rsp $'Script will now exit. Press any key to continue...\n' -n1 key
```
Comment (chaen) [inline tests/Jenkins/utilities.sh]: You do not want interactive actions on scripts executed by jenkins ;-)

Comment (vipersec) [inline tests/Jenkins/utilities.sh]: Oh, that's right.

**Comment by fstagni** [comment]
Review OK

**Pull request merged**